e2e/cypress/integration/enterprise/cloud/Billing/subscriptions_spec.js

```diff
@@ -0,0 +1,250 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @cloud_only
+
+describe('System Console - Subscriptions section', () => {
+    before(() => {
+        // * Check if server has license for Cloud
+        cy.apiRequireLicenseForFeature('Cloud');
+
+        cy.apiInitSetup().then(({team}) => {
+            cy.visit(`/${team.name}/channels/town-square`);
+        });
+
+        cy.intercept('**/api/v4/cloud/subscription').as('forNonExistanceOfTrailTag');
+        gotoSubscriptionScreen();
+    });
+
+    beforeEach(() => {
+        cy.intercept('**/api/v4/cloud/subscription').as('forVisibilityOfTrailTag');
+        cy.visit('/admin_console/billing/subscription');
+    });
+
+    it('MM-37054 - Non Existance of Trail tag on Subscription screen', () => {
+        // * Check for non existance of Trail tag
+        cy.wait('@forNonExistanceOfTrailTag').then((resObj) => {
+            expect(resObj.response.statusCode).to.equal(200);
+            const modifiedRes = {...resObj.response.body, ...resObj.response.body.is_free_trial = 'false'};
+            cy.intercept('**/api/v4/cloud/subscription', modifiedRes).as('nonExistanceTrailTag');
+            cy.contains('span', 'Trial').should('not.exist');
+            cy.wait('@nonExistanceTrailTag', {timeout: 5000}).then((xhr) => expect(xhr.response.statusCode).to.equal(200));
+        });
+    });
```

Comment:
Please move this to the new spec file where we'll be testing paid-tier. Also, there may be no point to mock the API since we'll be testing against a known paid-tier or free-trial server installation.

Reply:
renamed file and kept only free trial test cases in it and no mocking used anywhere in the script now.